dbtwiz/admin/__init__.py

```diff
@@ -0,0 +1,37 @@
+import typer
+from typing_extensions import Annotated
+
+from dbtwiz.target import Target
+
+from .cleanup import (
+    cleanup_development_dataset,
+    cleanup_orphaned_materializations,
+)
+
+
+app = typer.Typer()
+
+@app.command()
+def orphaned(
+        target: Annotated[Target, typer.Option(
+            "--target", "-t",
+            help="Target")] = Target.dev,
+        list_only: Annotated[bool, typer.Option(
```

Comment:
It isn't entirely clear which of `list_only` and `force_delete` that win if both are used, since they will then contradict one another. Might want to address this in `help` and/or give a warning to the user immediately saying that it's invalid to use both simultaneously

Reply:
Fixed by logging an error if you try to use both.